config/database.yml

```diff
@@ -5,14 +5,14 @@ default: &default
 
 development:
   <<: *default
-  database: mars_curiosity_development
+  database: mars_perseverance_development
```

Comment:
These changes and the change to `.travis.yml` aren't necessary. This is just the name of the database, and it would be very disruptive to need to rename the database or create a new one when this gets deployed.